source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1266,6 +1267,28 @@ message TransformationPermutePhiOperands {
 
 }
 
+message TransformationPropagateInstructionDown {
+
+  // Propagates an instruction from |block_id| into its successors.
+  // Concretely, the transformation clones the propagated instruction
+  // into some of the predecessors of |block_id| and removes the original
```

Comment:
Should "predecessors" be "successors" here?  And why only some of them?  Because it's only needed in the ones where it may be used?  In that case I'd say "into the necessary successors".

Reply:
Fixed in 69b41e9.